release-team/release-team-selection.md

```diff
@@ -0,0 +1,99 @@
+
+Moving to a [questionnaire](#sample-questionnaire) / survey format will allow us to capture more of the information required to vet potential shadows and well as begin to generate metrics around some of the Release Team processes.
+
+Questionnaire creation will be facilitated by SIG Release Chairs or a delegated coordinator to ensure consistency across release cycles.
+
+Following submissions, the questionnaires will be reviewed by role leads of the incoming Release Team.
+
+Volunteers that meet the requirements of the respective role will be contacted by those leads via some convenient mechanism (Slack, video chat, etc.) to further discuss.
+
+After vetting the volunteers for their roles, role leads should make a final decision on selected shadows with the incoming Release Team Lead.
+
+
+### Sample Questionnaire
+
+Thank you for volunteering for the Kubernetes 1.14 Release Team! 
+
+This questionnaire is meant to learn a little bit more about you, your journey with Kubernetes, your understanding of the Release Team processes and procedures, as well as to determine if you're a good fit for the current Release Team. Please note that you are expected to be inexperienced in some areas, as a role shadow is a mentorship opportunity.
+
+#### Logistics
+
+- What [Release Team role(s)](https://git.k8s.io/sig-release/release-team#kubernetes-release-team-roles) are you interested in shadowing? _[Select up to two (2)]_
+- The [Role Handbooks](https://git.k8s.io/sig-release/release-team/role-handbooks) defines the tasks and responsibilities of each role for this release cycle. Are you prepared to assist in fulfilling these duties? Please read the handbook relevant to the role you're interested in before answering.
+- Shadowing Release Team roles requires a substantial time commitment, especially during the periods specified in your role handbook. The time requirements specified are estimates and depending on the need, you may need to dedicate more time than initially estimated. This may include attending video meetings at inconvenient times. Is this time you can commit to?
+- Do you have any schedule conflicts during [this release cycle](https://git.k8s.io/sig-release/releases/release-1.14/README.md), such as vacations, school, or intense periods of work, that would make you unavailable for more than a couple of workdays?
+- The purpose of shadowing is to train new Release Team members. Assuming that your job or life situations don't change between now and then, are you willing to volunteer for the relevant role Lead position for 1.15 or 1.16? This usually requires support from your employer, as you will probably need to do release work during work hours.
+- What time zone are you usually in?
+
+#### Experience
```

Comment:
I'd like to see a little more pull on critical thinking from the person filling out the questionnaire asking them to describe why they're interested in the role in terms of specific past relevant experiences OR specific desired new experience they'd expect to gain from the learning/growth opportunity they see in the role.